source/JSONFaceApp.mc

```diff
@@ -17,8 +20,20 @@ class JSONFaceApp extends Application.AppBase {
     }
 
     // Return the initial view of your application here
-    function getInitialView() as Array<Views or InputDelegates>? {
```

Comment:
I love learning. What is the difference between these two syntaxes? Which is considered as best practice?

Reply:
Well it was required for my SDK version to even compile your code. I took the new signature from SDK generated example project. If your compiler complains about the new signature fell free to change it back.

To be honest this is my first time working with garmin SDK as I have bought my vivoactive5 in February this year. But I have played with it a bit to create new features and adjust your code to fit my needs. So implementing configuration for your code was a bit of a learning adventure for me. And I would say that the SDK is not that well documented.